### ISSUE TITLE
Reg\Events - External user can create a Restart report

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#2106]
#### Dependencies

- Blocked by
- Blocking

#### Description:

Brief description of the work involved, including any user research context or key decisions.

#### Acceptance Criteria:

Given I am authenticated external user with an SFO that has been closed or temporarily shutdown
When I click on the 'Restart' button
Then I am redirected to a new page to fill out a Restart form (specific to SFO restart)
[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14898-337&t=jRrigJgcVZK8lUv4-1)

Given I am authenticated external user with an LFO that has been closed or temporarily shutdown
When I click on the 'Restart' button
Then I am redirected to a new page to fill out a Restart form (specific to LFO restart)
[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14898-337&t=jRrigJgcVZK8lUv4-1)

Given I am authenticated external user with a facility that has been closed or temporarily shutdown
When I click on the 'Restart' button
Then I am redirected to a new page to fill out a Restart form (specific to facility restart)
[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14898-337&t=jRrigJgcVZK8lUv4-1)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner
